Chessington.GameEngine/Pieces/Moves.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Chessington.GameEngine.Pieces {
+    public class Moves {
+        public static IEnumerable<Square> GetLateralMoves(Board board, Square here)
+        {
+            List<Square> availableMoves = new List<Square>();
+            foreach (var index in Enumerable.Range(0,8))
```

Comment:
Ok, nice one. Since we are starting to cleaning classes and methods up, we should take care of this magic numbers.

Magic numbers are just numbers in your code that don't have explanation. This numbers can also change.
You should assign those numbers into constants.

In your case you already have this constant in GameSettings class.